### PR TITLE
Improving usage chart and piechart tooltips formatting

### PR DESCRIPTION
**Pull request by bitamar**
#629 #586 

screen shot 2015-07-21 at 5 28 53 pm

### CODE REVIEW

File: client/app/scripts/filters/chart-usage-dataset.filter.js [hunk 1]
```diff
@@ -138,11 +139,11 @@ angular.module('negawattClientApp')
         angular.forEach(values, function(item, timestamp) {
           var label = moment.unix(timestamp).format(frequency.axis_h_format);
           var col = [
-            { 'v': label },
-            { 'v': item.flat },
-            { 'v': item.peak },
-            { 'v': item.mid  },
-            { 'v': item.low  }
+            {v: label},
+            {v: item.flat, f: $filter('number')(item.flat, 0) + ' קוט״ש'},
```
Comment (bitamar) [inline client/app/scripts/filters/chart-usage-dataset.filter.js]: Should be changed to "קילוואט", according to selected frequency

File: client/app/scripts/filters/chart-usage-dataset.filter.js [hunk 2]
```diff
@@ -134,16 +135,16 @@ angular.module('negawattClientApp')
           prevRateType = item.rate_type;
         });
 
+        // Display the unit according to selected frequency.
+        var unit = ['hour', 'minute'].indexOf(frequency.frequency) != -1 ? 'קו״ט' : 'קוט״ש';
```
Comment (bitamar) [inline client/app/scripts/filters/chart-usage-dataset.filter.js]: @shahardo - Should the unit label be "קו״ט" or "KW"?

**Comment by bitamar** [comment]
screen shot 2015-07-26 at 9 11 26 am
screen shot 2015-07-26 at 9 11 16 am

File: client/app/scripts/filters/chart-pie-dataset.filter.js [hunk 3]
```diff
@@ -78,7 +79,7 @@ angular.module('negawattClientApp')
             this.push({
               c: [
                 {v: !Utils.isEmpty(labels) && labels[key].label || 'category ' + key},
-                {v: +value},
+                {v: +value, f: $filter('number')(value, 0) + ' קוט״ש'},
```
Comment (amitaibu) [inline client/app/scripts/filters/chart-pie-dataset.filter.js]: add comment.

**Comment by bitamar** [comment]
Merged

**Pull request closed without merging**